src/citrine/resources/file_link.py

```diff
@@ -405,7 +405,7 @@ def _complete_upload(self, dest_name: str, uploader: _Uploader):
         url = self._get_path(file_id) + '/versions/{}'.format(version)
         return FileLink(filename=dest_name, url=url)
 
-    def download(self, file_link: FileLink, local_path: str):
+    def download(self, file_link: FileLink, *, local_path: str):
```

Comment:
Both arguments are used and one is not necessary primary, so I would make them both keyword-args